mesonbuild/arglist.py

```diff
@@ -257,7 +257,7 @@ def extend_preserving_lflags(self, iterable: T.Iterable[str]) -> None:
         normal_flags = []
         lflags = []
         for i in iterable:
-            if i not in self.always_dedup_args and (i.startswith('-l') or i.startswith('-L')):
+            if i not in self.always_dedup_args and (i.startswith(('-l', '-L'))):
```

Comment:
The parentheses around i.startswith is now redundant.

Reply:
I am unsure why you think this PR will never be merged...

Please squash "fixup" commits, same as https://github.com/mesonbuild/meson/pull/13478#discussion_r1694045716